Spec/TestUtilities.swift

```diff
@@ -43,6 +43,7 @@ let appSetupJson = JSON(parseJSON: try! String(contentsOfFile: pathForTestResour
 
 let testTimeout: TimeInterval = 10.0
 let testResourcesPath = "ably-common/test-resources/"
+let echoServerAddress = "https://shrouded-plains-50367.herokuapp.com/createJWT" // TODO: change this
```

Comment:
You can just deploy your changes to the Heroku echoserver, even if not fully merged, to avoid having to go back and do this.

Reply:
Just didn't want to disrupt existing stuff that currently depends on the echo server. Happy to change this when the echo server PR is merged.

Reply:
But your changes aren’t breaking changes? You only added a new endpoint so I don’t see the risk personally

Reply:
Only potential issue could be https://github.com/ably/echoserver/pull/4/commits/49a3a0f8af59a02a18f0c36451c958cfcf6224d0

Reply:
@funkyboy why would that be an issue?  I appreciate it's an upgrade, but you have run a full test suite against the echoServer running the new Node without any issues?

Reply:
No. Will do that with the Ruby SDK. If everything passes can I deploy then to echo.ably.io ?

Reply:
I ran the full test suite of Ruby against the echo server in staging. All green: https://travis-ci.org/ably/ably-ruby/builds/388329095

Will promote it to production if there are no objections.

Reply:
👍